components/_patterns/01-atoms/02-text/00-headings/heading-1/heading-1.twig

```diff
@@ -1,17 +1,19 @@
 {#
 /**
  * Available variables:
+ * - h1_base_class - the base class name
  * - heading_1_modifiers - array of modifiers to add to the base classname
  * - heading_1_blockname - blockname prepended to the base classname
  * - heading_1_url - (optional) the url the heading should poing to
  * - heading_1 - the content of the heading (typically text)
  */
 #}
-<h1 class="heading-1{% for modifier in heading_1_modifiers %} heading-1--{{ modifier }}{% endfor %}{% if heading_1_blockname %} {{ heading_1_blockname }}__heading-1{% endif %}">
+<h1 class="{{ h1_base_class|default('heading-1') }}{% for modifier in heading_1_modifiers %} {{ h1_base_class|default('heading-1') }}--{{ modifier }}{% endfor %}{% if heading_1_blockname %} {{ heading_1_blockname }}__{{ h1_base_class|default('heading-1') }}{% endif %}">
   {% if heading_1_url %}
     {% include "@atoms/01-links/link/link.twig"
       with {
-        "link_blockname": "heading-1",
+        "link_base_class": heading_1_link_base_class,
```

Comment:
Is it intentional to lose the default styling if I change the base class?  If it is then great, this passes.  If it is an unexpected consequence then maybe we should look at this again. Example:

I changed the base class here to "test" and lost the default heading-1 styles.

I am also thinking that we should beef up the heading-1~linked.yml file to include some example usage / defaults.  I know the defaults are already passed by this code but the yml files would be easier to use (and modify in the future) if they included the defaults.